common/src/main/java/gov/cms/ab2d/common/service/JobServiceImpl.java

```diff
@@ -162,15 +169,25 @@ public Resource getResourceForJob(String jobUuid, String fileName, String organi
         return resource;
     }
 
+
+    @Override
+    public void incrementDownloadCount(File file, String jobUuid) {
+        Job job = jobRepository.findByJobUuid(jobUuid);
+        JobOutput jobOutput = jobOutputService.findByFilePathAndJob(file.getName(), job);
+        jobOutput.setDownloaded(jobOutput.getDownloaded() + 1);
```

Comment:
I'm worried about concurrency here. There's not normally multiple users downloading the same file but it could happen.

Reply:
Simply put in a comment about what would happen if multiple parallel downloads were attempted.  Laila, what are your thoughts?